rapids-cmake/cpm/thrust.cmake

```diff
@@ -90,18 +97,27 @@ function(rapids_cpm_thrust NAMESPACE namespaces_name)
   set(multi_value)
   cmake_parse_arguments(_RAPIDS "${options}" "${one_value}" "${multi_value}" ${ARGN})
 
+  set(post_find_code "if(NOT TARGET ${namespaces_name}::Thrust)"
+                     "  thrust_create_target(${namespaces_name}::Thrust FROM_OPTIONS)" "endif()")
```

Comment:
I don't believe that this should break any downstream consumer that has a proper `<pkg>-config.cmake`. Since `<pkg>-config.cmake` can be called numerous times calls to `thrust_create_target` are wrapped inside a check for the target e.g.:
```cmake
if(NOT TARGET <pkg>::Thrust)
  thrust_create_target(<pkg>::Thrust FROM_OPTIONS)
endif()
```

So if we do break some downstream consumers, those consumer are already writing out a bad config file and need a change.

Reply:
Got it.